### ISSUE TITLE
Accuracy of CIFAR-FS

### ISSUE THREAD

**Issue by SivanDoveh** [kjunelee/MetaOptNet#24]
Hi, I downloaded your code and run it on CIFAR-FS with this line:
python train.py --gpu 0 --save-path "./experiments/CIFAR_FS_MetaOptNet_RR" --train-shot 5 \
--head Ridge --network ResNet --dataset CIFAR_FS

but I got 'best accuracy' of 78%. What can I do in order to get 84%?

Thank you!
Sivan

**Comment by kjunelee** [comment]
Hello. I have just finished a new experiment on a new server and the accuracies on CIFAR_FS are:
77.96 ± 0.38 % (meta-validation set)
84.34 ± 0.49 % (meta-test set).
You might want to check if you ran your test code on meta-test set as opposed to meta-validation set.

**Comment by SivanDoveh** [comment]
Thanks, it worked. 
Do you know why there is such a difference between the test and the validation set?

**Comment by kjunelee** [comment]
My intuition is that each split (train, test and validation) contains different categories and the result depends on the distance between training categories and test categories.
